prime-router/src/test/kotlin/FileSettingsTests.kt

```diff
@@ -94,7 +94,7 @@ class FileSettingsTests {
                 countyName = null,
                 senders = listOf(),
                 receivers = listOf(
-                    Receiver("elr", "single", "topic", CustomerStatus.INACTIVE, "schema")
+                    Receiver("elr", "single", "covid-19", CustomerStatus.INACTIVE, "schema")
```

Comment:
Moving the enum to SettingsProvider so it can be used by both, but not updating Receiver as part of this PR since it is sender-focused.  Updating Receiver to use the enum value would take additional work and testing changes.

Reply:
Tech Debt ticket: https://app.zenhub.com/workspaces/pipeline-6182b02547c1130010f459db/issues/cdcgov/prime-reportstream/5282